src/DbSelect.php

```diff
@@ -103,7 +105,7 @@ public function getItems($offset, $itemCountPerPage)
      * @return int
      * @throws MissingRowCountColumnException
      */
-    public function count()
+    public function count(): int
```

Comment:
Please remove all changes like this because the classes are not final and this means a BC break.
Thanks in advance! 👍